test/blackbox-tests/test-cases/fmt/run.t

```diff
@@ -85,3 +85,29 @@ In multi-line strings, newlines are escaped.
   (echo "multi\nline\nstring\n")
   
   (echo "multi\nline\nstring\n")
+
+Comments are preserved.
+
+  $ printf '; comment\n(a b;comment\nc;multi\n;line\n;comment\n d)' | dune unstable-fmt
```

Comment:
https://github.com/ocaml/dune/pull/1767 should make this test easier to read

Reply:
indeed, thanks!